packages/compass-data-modeling/src/components/diagram-card.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  Card,
+  css,
+  cx,
+  ItemActionMenu,
+  palette,
+  spacing,
+  Subtitle,
+  useDarkMode,
+} from '@mongodb-js/compass-components';
+import type { MongoDBDataModelDescription } from '../services/data-model-storage';
+import React from 'react';
+
+// Same as saved-queries-aggregations
+export const CARD_WIDTH = spacing[1600] * 4;
+export const CARD_HEIGHT = 218;
+
+const diagramCardStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const cardHeaderStyles = css({
+  display: 'flex',
+  gap: spacing[200],
+  alignItems: 'flex-start',
+});
+const cardTitle = css({
+  fontWeight: 'bold',
+  height: spacing[600] * 2,
+  marginBottom: spacing[400],
+
+  // WebkitLineClamp css property is in a very weird state in the spec and
+  // requires using deprecated flexbox spec props, but this does work in
+  // (Chromium) Electron and is the only way to get multiline text overflow to
+  // work using CSS only
```

Comment:
We don't only support chrome though, can we do without it and do text overflow for a single line?

Reply:
yeah, i'd do that - this file is mostly copy/paste of saved-queries-item card!

Reply:
Yeah, got it! At some point I think we might need to drop it there too when support for this feature lands in web